### DISCUSSION TITLE
Welchen API-Key (command) um das Laden zu starten?

### DISCUSSION THREAD

**Discussion by christiandulle** [goecharger/go-eCharger-API-v2#183] [Q&A]
ich möchte gerne die cloud api nutzen um die wallbox mit solar überschuss laden zu lassen. die home assistant integration ist unbrauchbar.

kann schon den ladestrom steuern mit "https://serial.api.v3.go-e.io/api/set?token=TOKEN&amp=7
wie kann ich tatsächlich das laden starten, wie mit dem großen runden knopf auf der app?

Ideal wäre natürlich eine vernünftige Einbindung der Cloud API in Home Assistant.

LG

**Comment by 0xFEEDC0DE64** [comment]
der große runde button macht mehrere dinge auf einmal (transaktion starten, forceState setzen, randomDelays beenden, ...), am einfachsten wäre es, wenn man weiß, warum es gerade nicht lädt (modelStatus api key verrät das)

Die Frage lautet ja, wie man die Ladung startet, warum lässt den der Charger in dem Moment nicht laden?

Wurde der Modus verstellt, oder ist authentifizierung notwendig, oder wurde der ForceState auf ForceOff gestellt?

**Reply by K3RL** [reply]
Wenn ich den existierenden Thread hier kurz hijacken darf?

Welche Cloud API Parameter kann ich setzen, um die Funktion des großen Buttons am Homescreen der App (Labels "Fortsetzen im Basic-Modus" etc.) zu replizieren und in jeder denkbaren Situation (insb. - Laden via App gestoppt - oder - gerade konnektiert und noch nicht authentifiziert -) das Laden zu starten? Würde gerne über die Apple Shortcuts-Funktion ein Widget erstellen, mit welchem ich das Laden zuverlässig freigeben kann (über einen GET-Request) Das ginge schneller und unkomplizierter als über die App, die zwar sehr gut ist und stabil funktioniert, aber zeitweise lange lädt, wenn sie geschlossen wurde.

Wie weit ich aktuell wäre:
frc = 0 (für Ladestart); frc = 1 (für Ladestopp)
trx = 0 (bzw. 1 äquivalent zu RFID Tag 0 etc...) bei Ladestart; trx=0 (bei Ladestopp, wenn mit RFID-Äquivalent authentifiziert wurde)

Edit: Es funktioniert seit Monaten zuverlässig; und zwar setze ich via „Homebridge Http Switch-Plugin“ (klarerweise über Homebridge am Raspberry PI) „trx=1&frc=0“ für den Ladestart und „frc=1“ für den Stopp. Zudem verweist StatusURL des Plugins auf die API-Abfrage  „filter=alw“. Damit habe ich die Wallbox in Homekit zur vollen Zufriedenheit integriert.
